projects/dotcms-field-elements/src/components/dot-date-time/dot-date-time.e2e.ts

```diff
@@ -1,140 +1,446 @@
+
+                await page.waitForChanges();
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    value: '2019-01-01 10:10:10'
+                });
+                expect(spyValueChangeEvent.events.length).toEqual(1);
+            });
+
+            it('should emit status and value on Reset', async () => {
+                await element.callMethod('reset');
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: true,
+                        dotTouched: false,
+                        dotValid: true
+                    }
+                });
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    value: ''
+                });
+            });
+
+            it('should send status and value change and stop dot-input-calendar events', async () => {
+                const evt_statusChange = await page.spyOnEvent('_statusChange');
+                const evt_valueChange = await page.spyOnEvent('_valueChange');
+
+                dateInput.triggerEvent('_valueChange', {
+                    detail: {
+                        name: '-date',
+                        value: '2019-01-01'
+                    }
+                });
+                timeInput.triggerEvent('_valueChange', {
+                    detail: {
+                        name: '-time',
+                        value: '10:10:10'
+                    }
+                });
+                dotTestUtil.triggerStatusChange(false, true, true, timeInput);
+
+                await page.waitForChanges();
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: false,
+                        dotTouched: true,
+                        dotValid: true
+                    }
+                });
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    value: '2019-01-01 10:10:10'
+                });
+
+                expect(evt_statusChange.events.length).toEqual(0);
+                expect(evt_valueChange.events.length).toEqual(0);
+            });
+        });
+
+        describe('status change', () => {
+            it('should send status when dot-input-calendar send it', async () => {
+                dotTestUtil.triggerStatusChange(false, true, false, timeInput);
+                await page.waitForChanges();
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: '',
+                    status: {
+                        dotPristine: false,
+                        dotTouched: true,
+                        dotValid: false
+                    }
+                });
+            });
```

Comment:
Add the tests for the required, load element to emit the right statusChange, I merged the branch.